src/axios.ts

```diff
@@ -75,15 +75,15 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
   const Proto = Object.getPrototypeOf(axiosInstance);
 
   Proto.logout = function(): void {
-    this.get('/user/logout');
+    this.get('user/logout');
```

Comment:
Trailing `/`?

Reply:
I actually can't find this route in the API and the url in the code doesn't work at all (trailing slash or not)